src/main/java/com/asusoftware/mytransporter/model/Company.java

```diff
@@ -22,19 +20,16 @@
 
     @NotNull
     @Column(name = "company_name", nullable = false)
-    private String companyName;
-
-    @Email
-    @NotBlank
-    @Column(name = "email", nullable = false)
-    private String email;
+    private String name;
 
     @NotNull
     @Column(name = "address", nullable = false)
     private String address;
 
-    // Employee
-    @OneToMany(mappedBy = "company")
-    private List<User> userList;
+    // @TODO add price per kg
+
+    @OneToMany(cascade = CascadeType.DETACH)
+    @JoinColumn(name="couriers_id")
+    private List<User> couriers;
```

Comment:
ce e prostia asta? cum poate o lista sa aiba o coloana asignata (couriers_id)?

Reply:
Pai cum fac? E faza ca daca nu puneam asa, si bossul de la companie, tot in curiers era pus